tools/WinObjC.Tools/WinObjC.Tools.nuproj

```diff
@@ -14,19 +14,18 @@
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Configuration">
-    <PackageId>WinObjC.Tools</PackageId>
-    <Title>WinObjC.Tools</Title>
+    <PackageId>winobjc-tools</PackageId>
+    <Title>winobjc-tools</Title>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
-    <Summary>WinObjC.Tools</Summary>
-    <Description>WinObjC.Tools</Description>
+    <Summary>Command Line tools and Visual Studio Extensions for the WinObjC Project.</Summary>
+    <Description>Command Line tools and Visual Studio Extensions for the WinObjC Project. This package includes vsimporter, the tool to convert an existing xcode objective c project to a visual studio solution, as well as the Visual Studio Extensions for Objective C syntax highlighting and Nuget package creation. See https://github.com/Microsoft/WinObjC for more details.</Description>
     <PackageReleaseNotes>
     </PackageReleaseNotes>
```

Comment:
We should note here somewhere that this package is for VS 2017. Maybe compatibility notes specifically go in the PackageReleaseNotes section? At the very least change occurrences of Visual Studio to Visual Studio 2017 in the summary & description.